msc_pygeoapi/provider/hrdps_zarr.py

```diff
@@ -448,6 +493,45 @@ def _gen_domain_axis(self, data):
     return aa, all_dims
 
 
+def _make_where_str(first_query: dict, new_query: dict):
+    """
+    Helper function to make a where query string for
+    nearest method when exact coordinates are not found
+    :param first_query: original_dict of query
+    :param new_query: transformed_dict of query
+    :returns: where query string for use in .where()
+    """
+
+    where_str = ''
+    for key in first_query.keys():
+        if key not in new_query.keys():
+            where_str += f'(self._data.{key}>={first_query[key].start}) & '
+            where_str += f'(self._data.{key}<={first_query[key].stop}) & '
+    LOGGER.debug(f'Where query string: {where_str}')
+    return where_str[:-3]
+
+
+def _convert_subset_to_crs(new_lat: slice, new_lon: slice, crs):
```

Comment:
Provide a return type in the function annotation, so:

```python
def _make_where_str(first_query: dict, new_query: dict) -> Tuple:
```

In this case you will have to add the following import:

```python
from typing import Tuple
```